BAZProject/SearchView/SearchCollectionViewCell.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  SearchCollectionViewCell.swift
+//  BAZProject
+//
+//  Created by avirgilio on 02/02/23.
+//
+
+import UIKit
+
+class SearchCollectionViewCell: UICollectionViewCell {
```

Comment:
Agrega un TODO que haga referencia al trabajo futuro que debes hacer sobre esta clase, o en caso de no utilizarla, mejor borrar la clase.

```suggestion
// TODO: Describir lo que debes trabajar
class SearchCollectionViewCell: UICollectionViewCell {
```